app/src/main/java/com/sunasterisk/rememory/ui/launch/LaunchActivity.kt

```diff
@@ -1,13 +1,14 @@
 package com.sunasterisk.rememory.ui.launch
 
+import android.content.Intent
 import android.os.Bundle
 import androidx.appcompat.app.AppCompatActivity
-import com.sunasterisk.rememory.R
+import com.sunasterisk.rememory.ui.main.MainActivity
 
 class LaunchActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_launch)
+        startActivity(Intent(this, MainActivity::class.java))
```

Comment:
conventions 2.11 
Viết hàm getIntent trong class MainActivity trả về intent hướng tới MainActivity 
ở đây chỉ gọi 
```suggestion
        startActivity(MainActivity.getIntent())
```